server/src/dtos/search.dto.ts

```diff
@@ -84,8 +84,6 @@ class BaseSearchDto {
   @Optional()
   state?: string;
 
-  @IsString()
-  @IsNotEmpty()
```

Comment:
You need to add these back. The next line needs to be `@Optional({ nullable: true })` instead.